ProjectCoimbra.UWP/Project.Coimbra.Midi/Models/InstrumentInfo.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed under the MIT License.
+
+namespace Coimbra.Midi.Models
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using Melanchall.DryWetMidi.Common;
+    using Melanchall.DryWetMidi.Standards;
+
+    /// <summary>
+    /// A class representing an instrument.
+    /// </summary>
+    public class InstrumentInfo
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InstrumentInfo"/> class.
+        /// </summary>
+        /// <param name="channel">Channel of the instrument</param>
+        /// <param name="programNumbers">Program numbers of the instrument</param>
+        /// <param name="noteCount">Note count of the instrument</param>
+        public InstrumentInfo(FourBitNumber channel, ICollection<SevenBitNumber> programNumbers, int noteCount)
+        {
+            Channel = channel;
+            ProgramNumbers = programNumbers;
+            NoteCount = noteCount;
+        }
+
+        /// <summary>
+        /// A string that contains the name and the note count of the instrument
+        /// </summary>
+        public string NameAndNoteCount => string.Format("{0} - {1} notes", string.Join(
```

Comment:
Add `CultureInfo.UICulture` to this to make the culture clear.

I'd also suggest seeing if you can format the number with commas, to make it easier for people to read if there's a large number.